tests/api_workflow/test_api_workflow_datasets.py

```diff
@@ -5,61 +5,124 @@
 
 class TestApiWorkflowDatasets(MockedApiWorkflowSetup):
 
-    def test_create_dataset_new(self):
+    def setUp(self, token="token_xyz", dataset_id="dataset_id_xyz") -> None:
+        super().setUp(token, dataset_id)
         self.api_workflow_client._datasets_api.reset()
```

Comment:
Removed all `self.api_workflow_client._datasets_api.reset()` calls in the test functions and added `setUp` instead.